encryption_works.md

```diff
@@ -299,6 +281,16 @@ Here's a message that was encrypted to my public key. Without having access to m
     =a8FR
     -----END PGP MESSAGE-----
 
+
+### The Web of Trust 
+
+PGP works best when a web of trust is created. Strong crypto can still be broken since humans are the weakest elements of any security system. People using PGP can fall victim to what's called a "man in the middle attack." Let's say you're talking to your colleague via encrypted e-mail. An adversity could potentially trick you into thinking that their public key was your colleagues. You'd encrypt a message with your private key and my public key. I can decrypt it, read and/or tamper with it, and then send the compromised message onto your college, with info purported to be from you. 
```

Comment:
not technically an MITM; it's a social engineering attack.
ALSO! point out that if you can, being able to verify the email came from the correct email server the address corresponds to helps mitigate such attacks.  (i.e. my email is "xx@gmail.com" but if i send you an email that comes from another SMTP server than google's, you should be cautious!)